src/components/screen/Calender/Calender.tsx

```diff
@@ -50,48 +50,55 @@ const Calendar: React.FC<CalenderProps> = ({ onChange }) => {
     setTargetMonth((current) => addMonths(current, 1))
   }
 
-  const handleSubmit = () => {
+  useEffect(() => {
     onChange(selectedDates)
-  }
+  }, [selectedDates])
 
   return (
-    <div className='mx-auto my-2 grid w-2/5 '>
-      <div className='my-2 flex items-center justify-center text-3xl'>
-        <button className='btn btn-ghost mr-auto text-xl' onClick={handlePreviousMonth}>
+    <div className='flex-col'>
+      <div className='flex items-center justify-center text-3xl'>
+        <button
+          className='btn btn-ghost mr-auto p-0 text-lg md:ml-auto md:text-xl'
```

Comment:
paddingないとボタンがキッツキツになってるので注意
```suggestion
          className='btn btn-ghost mr-auto p-2 text-lg md:ml-auto md:text-xl'
```

Reply:
[0f7c0bc](https://github.com/NUTFes/chosei-chan/pull/46/commits/0f7c0bc31f2f95b4d9d9a6d353cf93e51a31200b)